### PR TITLE
Common_CreateCustomException

### PR DESCRIPTION
**Pull request by Maksim-Trolina**

### CODE REVIEW

File: Common/CustomException/ForbiddenException.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (nmamro) [inline Common/CustomException/ForbiddenException.cs]: Do we use all these usings?

**Review by Cerberous007 — APPROVED** [review]
Don`t forget to remove useless usings and merge develop in your branch before merge

**Review by DanilFadeeev — APPROVED** [review]

**Review by denis-kriv — APPROVED** [review]

**Review by dTarasova — CHANGES_REQUESTED** [review]
Why don't we standardize messages in exceptions? 
I think we also should write in summary situations where we use these exceptions

**Comment by Maksim-Trolina** [comment]
> Why don't we standardize messages in exceptions?
> I think we also should write in summary situations where we use these exceptions


At this stage, this is enough for us then we will add our logic there, if necessary

**Review by studokim — APPROVED** [review]
Agree with Dasha about summary, but okay

File: Common/CustomException/BadRequestException.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    public class BadRequestException : Exception
+    {
+        public BadRequestException(string message)
+            : base(message)
+        { }
+
+        public BadRequestException()
+        { }
+
+        public BadRequestException(string message, Exception inner)
```
Comment (ifelseelif) [inline Common/CustomException/BadRequestException.cs:14]: why do we need this constructor with Exception inner ?

Comment (DarkFelas) [reply Common/CustomException/BadRequestException.cs:14]: Test slack integration

**Review by DarkFelas — CHANGES_REQUESTED** [review]
Test slack

**Review by DarkFelas — APPROVED** [review]

File: Common/CustomException/BadRequestException.cs (hunk 2 above)
Comment (ifelseelif) [reply Common/CustomException/BadRequestException.cs:14]: what does it mean? I didn't understand.

Comment (DarkFelas) [reply Common/CustomException/BadRequestException.cs:14]: Check telegram ))

**Review by ifelseelif — APPROVED** [review]

**Comment by nmamro** [comment]
Seems ok overall for me, waiting for Dasha

**Review by Dimentr01 — APPROVED** [review]

**Pull request closed without merging**